urgent-care/pom.xml

```diff
@@ -44,6 +39,11 @@
       <artifactId>spring-beans</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>gov.va.api.health</groupId>
+      <artifactId>ee-artifacts</artifactId>
+      <version>1.1.31</version>
```

Comment:
Can we extract these versions as variables and put them at the top of the POM?

Like its done in other repos (mr-anderson does it).

Just pull the versions out into properties.  Also why is it 1.1.31?  That seems super high.

Reply:
Can you do this for the:

    <dependency>
      <groupId>gov.va.api.health</groupId>
      <artifactId>r4</artifactId>
      <version>1.0.1</version>
    </dependency>

as well above.  I should have found that in the last PR.